index.html

```diff
@@ -32,23 +32,157 @@ <h3>
           <a href="#" title="Linked-in"><i class="fa-brands fa-linkedin-in no-decoration"></i></a>
         </li>
-        <li class="medium">
+        <li>
           <a href="#" title="Medium"><i class="fa-brands fa-medium no-decoration"></i></a>
         </li>
-        <li class="github">
+        <li>
           <a href="#" title="GitHub"><i class="fa-brands fa-github no-decoration"></i></a>
         </li>
-        <li class="angellist">
+        <li>
           <a href="#" title="AngelList"><i class="fa-brands fa-angellist no-decoration"></i></a>
         </li>
       </ul>
     </section>
+
+    <div class="works">
+      <article>
+        <a href="#"><img src="assets/work_1.png" alt="Preview of this project"></a>
+        <h2><a href="#">Tonic</a></h2>
+        <div class="info">
+          <span><a href="#">CANOPY</a></span>
+          <div class="dot"></div>
+          <span><a href="#">Back End Dev</a></span>
+          <div class="dot"></div>
+          <span><a href="#">2015</a></span>
+        </div>
+        <h3 class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.</h3>
+        <ul class="tags">
+          <li><a href="#">html</a></li>
+          <li><a href="#">css</a></li>
+          <li><a href="#">javaScript</a></li>
+        </ul>
+        <button type="button" class="btn-more">See Project</button>
+      </article>
+
+      <article>
+        <a href="#"><img src="assets/work_2.png" alt="Preview of this project"></a>
+        <h2><a href="#">Multi-Post Stories</a></h2>
+        <div class="info">
+          <span><a href="#">CANOPY</a></span>
+          <div class="dot"></div>
+          <span><a href="#">Back End Dev</a></span>
+          <div class="dot"></div>
+          <span><a href="#">2015</a></span>
+        </div>
+        <h3 class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.</h3>
+        <ul class="tags">
+          <li><a href="#">html</a></li>
+          <li><a href="#">css</a></li>
+          <li><a href="#">javaScript</a></li>
+        </ul>
+        <button type="button" class="btn-more">See Project</button>
+      </article>
+
+      <article>
+        <a href="#"><img src="assets/work_3.png" alt="Preview of this project"></a>
+        <h2><a href="#">Tonic</a></h2>
+        <div class="info"> 
+          <span><a href="#">CANOPY</a></span>
+          <div class="dot"></div>
+          <span><a href="#">Back End Dev</a></span>
+          <div class="dot"></div>
+          <span><a href="#">2015</a></span>
+        </div>
+        <h3 class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.</h3>
+        <ul class="tags">
+          <li><a href="#">html</a></li>
+          <li><a href="#">css</a></li>
+          <li><a href="#">javaScript</a></li>
+        </ul>
+        <button type="button" class="btn-more">See Project</button>
+      </article>
+
```

Comment:
- [ ]  Kindly wrap this article with a section tag for more semantic HTML as it is a section of the project